test/EFCore.PG.Tests/Metadata/Conventions/NpgsqlValueGenerationStrategyConventionTest.cs

```diff
@@ -29,9 +30,12 @@ public void Annotations_are_added_when_conventional_model_builder_is_used_with_s
                 .ForNpgsqlUseSequenceHiLo()
                 .Model;
 
+            model.RemoveAnnotation(CoreAnnotationNames.ProductVersion);
+
+            //throw new Exception("Annotations: " + string.Join(", ", model.GetAnnotations().Select(a => a.Name)));
```

Comment:
Why commented?

Reply:
Oops, that was a debugging line that was left in - removed it.